### PR TITLE
Azure - Add Batch and VM Scale Set resources

### PR DESCRIPTION
**Pull request by tbarlow12**

### CODE REVIEW

File: tools/c7n_azure/tests/templates/vmscaleset.json [hunk 1]
```diff
@@ -0,0 +1,263 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "IotHubs_cctest_iothub_name": {
```
Comment (aluong) [inline tools/c7n_azure/tests/templates/vmscaleset.json]: This template includes more things than VMSS

Comment (tbarlow12) [reply tools/c7n_azure/tests/templates/vmscaleset.json]: I'll get this one and the other template cleaned up

File: docs/source/azure/policy/resources/batch.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.. _azure_batch:
+
+Batch
+=====
+
+Filters
+--------
+- Standard Value Filter (see :ref:`filters`)
+    - Model: `BatchAccount <https://docs.microsoft.com/en-us/python/api/azure.mgmt.batch.models.batchaccount?view=azure-python>`_
```
Comment (stefangordon) [inline docs/source/azure/policy/resources/batch.rst]: These links are great 👍

**Comment by stefangordon** [comment]
The cleanup on batch arm template is good, but the VMSS one still deploys Cognitive services and other stuff - I'd try to clean it up so its just the VMSS so its easy to deploy/delete for unit tests.

**Pull request merged**